test/Microsoft.AspNet.Mvc.ModelBinding.Test/project.json

```diff
@@ -5,6 +5,8 @@
     "dependencies": {
         "Microsoft.AspNet.FileProviders": "1.0.0-*",
         "Microsoft.AspNet.Http.Core": "1.0.0-*",
+        "Microsoft.AspNet.Mvc":"6.0.0-*",
+        "Microsoft.AspNet.Mvc.Core":"6.0.0-*",
```

Comment:
move the integration tests somewhere else, preferably a new test project because existing ones contain unit or functional tests.

(integration tests focused on model binding and validation scenarios are still integration tests, not model binding unit tests.)